src/components/repository-profile.component.js

```diff
@@ -79,10 +79,33 @@ const styles = StyleSheet.create({
   icon: {
     paddingBottom: 20,
   },
+  languageInfo: {
+    flexDirection: 'row',
+    marginTop: 35,
+  },
+  languageInfoTitle: {
+    color: colors.white,
+    paddingLeft: 3,
+    fontSize: normalize(10),
+    ...fonts.fontPrimary,
+  },
 });
 
 export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
   <View style={styles.container}>
+    {repository.language !== null &&
```

Comment:
From the doc (https://developer.github.com/v3/repos/#get) language can be `null` :)